napari/layers/_image_layer/model.py

```diff
@@ -391,6 +425,7 @@ def to_xml_list(self):
             as a png according to the svg specification.
         """
         image = np.clip(self._image_view, self.clim[0], self.clim[1])
+        image = image - self.clim[0]
```

Comment:
Hmmm, hardcoding the clims... I'm not sure I love it, but either way, I think I would prefer not to repeat variable names, keeping everything functional:

```python
low, high = self.clim
image_clipped = np.clip(self._image_view, low, high)
if color_range != 0:
    image_normalized = (image_clipped - low) / (high - low)
else:
    image_normalized = image_clipped
```

(I don't see the need to subtract the clim if the range is 0, and indeed I think it might be the wrong thing to do.)

Reply:
fixed the repetition of variable names and subtraction of clim for range `0`